.github/workflows/release-sdk-client.yml

```diff
@@ -0,0 +1,164 @@
+# This workflow exists because the client sdk build process requires macos.
+name: Manual Publish SDK Client Package
```

Comment:
nit: As we use it with workflow dispatch we may want to remove "manual".

Reply:
Leaving this to avoid creating another row in the Github Actions page.  Clearing workflows isn't supported without running `gh` commands.